qa/full-cluster-restart/src/test/java/org/opensearch/upgrades/FullClusterRestartIT.java

```diff
@@ -1344,8 +1350,8 @@ public void testOperationBasedRecovery() throws Exception {
             if (minimumNodeVersion().before(Version.V_2_0_0)) {
                 settings.put(IndexSettings.INDEX_SOFT_DELETES_SETTING.getKey(), randomBoolean());
             }
-            createIndex(index, settings.build());
-            ensureGreen(index);
+            final String mappings = randomBoolean() ? "\"_source\": { \"enabled\": false}" : null;
+            createIndex(index, settings.build(), mappings);            ensureGreen(index);
```

Comment:
Sorry, nit-picking, seems like `ensureGreen(index);` should be moved to the next line

Reply:
Good nit-pick! Correcting typo